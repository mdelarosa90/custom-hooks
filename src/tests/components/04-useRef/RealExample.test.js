import React from 'react';
import {shallow} from 'enzyme';
import {RealExampleRef} from '../../../components/04-useRef/RealExampleRef';

describe('Pruebas en <RealExample />', () => {
    const wrapper = shallow(<RealExampleRef />);
    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
    })

    test('debe mostrar el componente <MultipleCustomHook', () => {
        wrapper.find('button').simulate('click');
        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true);
    })
})