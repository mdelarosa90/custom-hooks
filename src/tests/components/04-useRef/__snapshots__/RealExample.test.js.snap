// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <RealExample /> debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Real Ref
  </h1>
  <hr />
  <button
    className="btn btn-outline-dark mt-5"
    onClick={[Function]}
  >
    Show / Hide
  </button>
</div>
`;
